python/lsst/meas/modelfit/regularizedMoments/regularizedMomentsContinued.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+
+def makeGaussian(x, y, scale, muX, muY, varX, varXY, varY):
```

Comment:
Docstring?

Reply:
Btw, I find it hard to believe that this doesn't exist somewhere? At least in some form you could delegate to.

Reply:
added, and really this is the best way to get to an elliptical Gaussian function